### PR TITLE
fix: Markdownlint literal nbsp

### PR DESCRIPTION
**Pull request by nschonni**
### Motivation

Copied rule from upstream repo. These characters were mostly cleaned up there after the Markdown conversion, and this would also be enforced by Prettier in the future. This rule will also autofix any new issues, so I bundled in the current files it fixed. There were a few extra punctuation spaces I manually dropped

### CODE REVIEW

**Review by yin1999 — APPROVED** [review]
LGTM for l10n-zh

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja`.

**Review by caugner — APPROVED** [review]

**Review by cw118 — APPROVED** [review]
Looks good for fr, thank you!

**Review by Graywolf9 — APPROVED** [review]
LGTM  for `es` Thank you!

**Review by josielrocha — APPROVED** [review]
Looks got for `pt-BR`. Thank you a lot!

**Review by pje1740 — APPROVED** [review]
LGTM for `ko`! Thank you 👍

**Review by sashasushko — APPROVED** [review]
Approve from RU. Thank you @nschonni 🙏

**Comment by cw118** [comment]
Merging since it looks like all locales and core-dev have approved :)

File: files/fr/web/css/css_grid_layout/basic_concepts_of_grid_layout/index.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Pour que notre exemple ressemble vraiment à une grille nous devons ajouter des
```
Comment (Cantbelieveitsmh) [inline files/fr/web/css/css_grid_layout/basic_concepts_of_grid_layout/index.md:92]: #10148

**Pull request merged**